java/src/main/native/src/TableJni.cpp

```diff
@@ -1190,7 +1190,15 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Table_convertCudfToArrowTable(JNIEnv
   try {
     cudf::jni::auto_set_device(env);
     std::unique_ptr<std::shared_ptr<arrow::Table>> result(new std::shared_ptr<arrow::Table>(nullptr));
-    *result = cudf::to_arrow(*tview, state->column_names);
+    auto column_metadata = std::vector<cudf::column_metadata>{};
```

Comment:
I realize this could be out of place on this PR, but this is an ugly thing to force on users of cudf. I feel like there should be a version of cudf::to_arrow that does this work to prevent this copied to every place that is using cudf.